tls/s2n_cipher_suites.h

```diff
@@ -31,7 +31,13 @@
 #define S2N_KEY_EXCHANGE_ECC      0x04  /* Elliptic curve cryptography */
 
 #define S2N_MAX_POSSIBLE_RECORD_ALGS    2
-#define S2N_CIPHER_SUITE_COUNT          38 /* Kept up-to-date by s2n_cipher_suite_match_test */
+#if !defined(S2N_NO_PQ)
+#define S2N_PQ_CIPHER_SUITE_COUNT       2
```

Comment:
It would be nice to make this less fragile going forward. Currently everyone touching the cipher suite offerings needs to remember to update these values?